index.html

```diff
@@ -19,7 +19,7 @@
     .colourpicker {
       margin-top: 3px;
       margin-bottom: 20px;
-      width: 30%;
+      width: 300px;
```

Comment:
whoops, didn't mean to make these changes in the same commit as the minification. Arggh. OH well, I just tested the minification and it didn't change so I know I did it at the right time.